### ISSUE TITLE
Exclude Options Are not working

### ISSUE THREAD

**Issue by rizikolik** [inthepocket/npm-license-scraper#12]
Hey guys,
First of all thanks for this great package .
I worked it and 99% of my demands are handled.
When I wanted to exclude some packages I encountered with a problem.
So basically in the readme file:
--exclude [package|package,package,package]: Ignore certain packages from the check (e.g submodules, monorepo or private packages)
 But I managed to made it work  by using it like that:
 
 - `--exclude=package,package,package`: Ignore certain packages from the check (e.g submodules, monorepo or private packages).
Example :
```shell
$ npx npm-license-scraper --exclude=@reduxjs/toolkit or npx npm-license-scraper --exclude=@reduxjs/toolkit,lodash
```

Also for the single package excluding,I needed to update 
https://github.com/inthepocket/npm-license-scraper/blob/e10ca6f94086bb689e1d7b30da320180c84e05de/src/index.ts#L67
from :
` } else if (typeof flags.exclude === 'boolean') {`
to 
` } else if (typeof flags.exclude === 'string') {`
Because when we provide a single value to exclude option, its shown as a boolean but because its only true or false,we cant filter dependencies.Because we need a string for filtering dependencies correctly.

**Comment by robrechtme** [comment]
Hi! Sorry for the late response 🙏 Feel free to open a PR if this is still an issue.

**Comment by robrechtme** [comment]
Seems like a bug indeed: #16

**Comment by robrechtme** [comment]
🎉 This issue is fixed in [v2.3.1](https://github.com/inthepocket/npm-license-scraper/releases/tag/v2.3.1)
